lib/claude/process-manager.ts

```
@@ -0,0 +1,450 @@
+
+  private processMessageBuffer(sessionId: string, buffer: string): void {
+    const lines = buffer.split('\n');
+    
+    for (const line of lines) {
+      if (line.trim()) {
+        try {
+          const message: ClaudeMessage = JSON.parse(line);
+          this.handleClaudeMessage(sessionId, message);
+        } catch (error) {
+          // Incomplete JSON, will be processed in next buffer
+          continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)